build_projects/dotnet-cli-build/PublishTargets.cs

```diff
@@ -94,8 +94,20 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
                     // Copy the latest installer files
                     CopyBlobs($"{Channel}/Installers/{CliNuGetVersion}/", $"{Channel}/Installers/Latest/");
 
-                    // Generate the SDK Version text files
-                    List<string> versionFiles = new List<string>() { "win.x86.version", "win.x64.version", "ubuntu.x64.version", "rhel.x64.version", "osx.x64.version", "debian.x64.version", "centos.x64.version" };
+                    // Generate the CLI and SDK Version text files
+                    List<string> versionFiles = new List<string>()
+                    {
+                        "win.x86.version",
+                        "win.x64.version",
+                        "ubuntu.x64.version",
```

Comment:
I would like to update all of these to be versioned.  How do you all think about older install-dotnet's being able to consume newer drops?